app/models/manageiq/providers/base_manager/event_catcher.rb

```diff
@@ -32,5 +32,16 @@ def self.validate_config_settings(configuration = VMDB::Config.new("vmdb"))
 
       subclasses.each { |subclass_key| configuration.merge_from_template_if_missing(*(path + [subclass_key])) }
     end
+
+    validate_event_flooding_settings(configuration, :defaults)
+  end
+
+  def self.validate_event_flooding_settings(config, provider)
```

Comment:
Ah, I see, it's overriden in the subclass.
How about using the inheritance/defaults mechanism from the new config revamp?
I'm new to this  and can look into that, if you like.